File: src/main/java/pl/com/tegess/EmbeddedJetty.java
package pl.com.tegess;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

@SpringBootApplication
@ComponentScan("pl.com.tegess")
@Configuration
@EnableAutoConfiguration
public class EmbeddedJetty {

    public static void main(String[] args) {
        SpringApplication.run(EmbeddedJetty.class, args);
    }

}

File: src/main/scala/com/tegess/EmbeddedJetty.scala
package com.tegess

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.{EnableAutoConfiguration, SpringBootApplication}
import org.springframework.context.annotation.{ComponentScan, Configuration}
import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity

@SpringBootApplication
@ComponentScan(Array("com.tegess"))
@Configuration
@EnableAutoConfiguration
@EnableGlobalMethodSecurity(prePostEnabled=true)
class SampleConfig

object EmbeddedJetty extends App {
  SpringApplication.run(classOf[SampleConfig])
}
